packages/server/src/application/types/index.ts

```diff
@@ -351,6 +353,9 @@ export interface TestRequest {
 export interface Test {
   start: () => Promise<void>;
   request: TestRequest;
+  fixtures: {
```

Comment:
You have to create a nested namespace in application/index

Reply:
What do you mean? Why it can't be inside the Test namespace?

Reply:
It's ok that its inside test but it should be a new namespace (commented above).

Reply:
Just did it but without the `createNamespace`, why do we need it?

Reply:
Namespace provides a functionality needed for dynamic module initialisation.